packages/gatsby-plugin-cxs/src/gatsby-ssr.js

```diff
@@ -0,0 +1,13 @@
+import React from "react"
+import cxs from "cxs"
+
+exports.onRenderBody = ({ setHeadComponents }) => {
+  const css = cxs.css()
+  setHeadComponents([
```

Comment:
Do we need to implement `cxs.reset()` in the [docs](https://github.com/cxs-css/cxs#staticserver-side-rendering)

Reply:
Based on my knowledge of the `ssr` stage, the `ssr` is only triggered during `gatsby build` stage, right?

If it's correct, I think we don't need to reset it. Because the cache in cxs is the memory cache([ref](https://github.com/cxs-css/cxs/blob/master/src/index.js#L20)), it is always the new thread for each `gatsby build`.